docs/content/dagster-cloud/managing-deployments/alerts/pagerduty.mdx

```diff
@@ -0,0 +1,164 @@
+---
+title: "PagerDuty & Dagster Cloud | Dagster Docs"
+description: "Create alerts that make PagerDuty incidents based on events in Dagster Cloud."
+
+platform_type: "cloud"
+feature: "alerting"
+integration: "pagerduty"
+---
+
+# PagerDuty & Dagster Cloud
+
+<ApplicableDagsterProduct />
+
+Dagster Cloud supports [sending alerts](/dagster-cloud/managing-deployments/alerts) to PagerDuty, which will automatically create a PagerDuty incident.
+
+---
+
+## PagerDuty + Dagster Cloud integration benefits
+
+- Empower your team with real-time incident response by integrating Dagster Cloud and PagerDuty
+- Notify on-call responders when alerts are triggered in Dagster Cloud
+- Dramatically reduce downtime for issues related to asset materialization, asset checks, job runs, schedules, sensors, and code locations
+- Customize the urgency and routing of PagerDuty incidents based on the content of the Dagster Cloud event payload
```

Comment:
I'm not positive we'll support this yet- could we remove

Reply:
I think this is the relevant feature https://support.pagerduty.com/docs/event-orchestration#advanced-event-orchestration, I haven't gotten to play around with it yet